docs/platform_concepts/growth/metrics.md

```diff
@@ -109,19 +109,34 @@ This widget shows the funnel of users who are coming to the bot. The blue lines
 
 ----
 
-### Whatsapp consumption
+### WhatsApp comsumption
 
-> This is calculated for WhatsApp channel.
+In the WhatsApp comsumption widget, you can select a WhatsApp phone number from the dropdown to see the consumption for that specific number. If no number is selected, the data will be displayed for all the numbers combined.
 
-Following three categories are tracked for the selected WhatsApp phone numbers: 
-1. Business initiated conversations 
-2. User initiated conversations 
-3. Referral conversations 
+<img src="https://hackmd.io/_uploads/H1J0UVwwn.png" alt="drawing" width="50%"/> 
 
-You can select the WhatsApp numbers from the bottom of the widget. 
+The three tracked categories for WhatsApp phone numbers are:
 
-<img src="https://i.imgur.com/4BFEBKI.png" alt="drawing" width="50%"/> 
+| Category                         | Sub category    | Description |
+| -------------------------------- | --- | ----------- |
+| **Business-initiated conversations** |     |  Conversations initiated by the bot or business to provide information, send notifications, or engage with users based on predefined triggers or events.        |
+||Authentication | Indicates the business initiated the conversation with a message template categorized as AUTHENTICATION. |
```

Comment:
suggestion: Notifies that the business initiated the conversation using an AUTHENTICATION message template.